tests/functions_user/whois_test.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+*
+* @package testing
+* @copyright (c) 2020 phpBB Group
+* @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+*
+*/
+
+require_once dirname(__FILE__) . '/../../phpBB/includes/functions_user.php';
+
+class phpbb_functions_user_whois_test extends phpbb_functional_test_case
```

Comment:
Oh, probably my fault, it should extend `phpbb_test_case` instead.

Reply:
I getting blind reading so much of code ehhe.